lib/katello/tasks/repository.rake

```diff
@@ -71,6 +71,12 @@ namespace :katello do
     end
   end
 
+  desc "Re-import all docker manifests to populate labels and annotations."
+  task :import_docker_manifest_labels => ["dynflow:client", "check_ping"] do
```

Comment:
What do you think about saying `container manifest` instead of `docker manifest` ? While our code still refers to docker, I think the public facing stuff could all say container.